variable/include/scipp/variable/variable.h

```diff
@@ -167,6 +167,7 @@ class SCIPP_VARIABLE_EXPORT Variable {
   VariableConcept &data() & { return *m_object; }
 
   void setVariances(Variable v);
+  auto &underlying() const { return *this; }
```

Comment:
Should probably do this via a constexpr if or type trait solution rather than adding `underlying` to `Variable` class directly.